README.md

```diff
@@ -141,6 +147,83 @@ window.addEventListener('DOMContentLoaded', () => {
 });
 ```
 
+### Example: Using InertiaJS with React via Webpacker
```

Comment:
I think we also need an example of a setup with Vue js. 
Maybe this part we should move to `Wiki` and have it per JS library